### PR TITLE
Pgettext "L" as "liter" to differentiate it from "L" as "Light"

### PR DESCRIPTION
**Pull request by Night-Pryanik**
Also, pgettexted other volume units as well.

### CODE REVIEW

File: src/cata_utility.cpp [hunk 1]
```diff
@@ -193,7 +193,7 @@ const char *volume_units_abbr()
     if( vol_units == "c" ) {
         return _( "c" );
     } else if( vol_units == "l" ) {
-        return _( "L" );
+        return pgettext( "Abbreviation for liter", "L" );
```
Comment (codemime) [inline src/cata_utility.cpp]: Generic contexts (say, `"volume_unit"` and `"mass_unit"`) would be lots better than sporadic ones.

Comment (Night-Pryanik) [reply src/cata_utility.cpp]: Sorry, I don't quite understand what do you mean.

Comment (codemime) [reply src/cata_utility.cpp]: I meant specifying the same context for all the units:
```c++
    if( vol_units == "c" ) {
        return pgettext( "volume_unit", "c" );
    } else if( vol_units == "l" ) {
        return pgettext( "volume_unit", "L" );
    } else {
        return pgettext( "volume_unit", "qt" );
    }
```

Comment (AMurkin) [reply src/cata_utility.cpp]: Use "volume_unit" for volumes and "mass_unit" for masses.

**Pull request merged**